pfcpiface/telemetry.go

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright(c) 2020 Intel Corporation
+
+package main
+
+import (
+	"net/http"
+
+	pb "github.com/omec-project/upf-epc/pfcpiface/bess_pb"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+func getPctiles() []float64 {
+	return []float64{50, 75, 90, 95, 99, 100}
+}
+
+func makeBuckets(values []uint64) map[float64]float64 {
+	buckets := make(map[float64]float64)
+	for idx, pctile := range getPctiles() {
+		buckets[pctile] = float64(values[idx])
+	}
+	return buckets
+}
+
+//upfCollector provides all UPF metrics
+type upfCollector struct {
+	rx_packets *prometheus.Desc
+	rx_bytes   *prometheus.Desc
+	rx_dropped *prometheus.Desc
+
+	tx_packets *prometheus.Desc
+	tx_bytes   *prometheus.Desc
+	tx_dropped *prometheus.Desc
```

Comment:
Variable names changes (``rx_packets, rx_bytes, rx_dropped, tx_packets, tx_bytes, tx_dropped``). Remove underscore? VSCode is printing out some warnings.